bin/setup

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env ruby
```

Comment:
purpose of this file?

Reply:
Bhai, I am using this file to have setup scripts that can automate the initialization of the application.